src/scopes.ts

```diff
@@ -478,6 +482,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     description: 'View the consent manager configuration.',
     products: [TranscendProduct.ConsentManager],
   },
+  [ScopeName.ManageAssessments]: {
```

Comment:
@abrantesarthur you need something like this

Reply:
true